### PR TITLE
chore(codeowners): removed

### LINKED ISSUE #2897: [CODEOWNERS] List of codeowners is out of date
## Detailed description

> Describe in detail the issue you're having.

The list of folks in the `CODEOWNERS` file is out of date and should be updated:
https://github.com/carbon-design-system/carbon-website/blob/main/.github/CODEOWNERS

> What did you expect to happen? What happened instead? What would you like to see changed?

Should be updated to latest list of owners.

## Additional information

- We should figure out who should be the updated owners for some of these and add to this issue.

### PR DESCRIPTION
**Pull request by sstrubberg**
Closes #2897

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,32 +0,0 @@
...
-src/pages/community/patterns/login-pattern/* @lukefirth
-
-# Eliad Moosavi and Natasha Decoste are the code owners for Data Visualization
-src/pages/data-visualization/* @theiliad @natashadecoste
-
-# Cal Smith for Angular tutorial
-/src/pages/tutorial/angular/* @zvonimirfras
-
-# Lee Chase for Vue tutorial
-/src/pages/tutorial/vue/* @lee-chase
```
Comment (tay1orjones) [inline .github/CODEOWNERS:32]: This part also needs to stay

**Comment by Huixh111** [comment]
o

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,32 +0,0 @@
...
-# Vincent Snagg is code owner for Experimental Watson Health patterns
-src/pages/community/patterns/chatbot/* @vincentsnagg
-
-# Adriana Morales is code owner for Experimental Cloud PAL patterns
-src/pages/community/patterns/order-summary-template/* @adrianamorales
-src/pages/community/patterns/cards/* @adrianamorales
-src/pages/community/patterns/dashboard-widgets/* @adrianamorales
-
-# Chris Reckling is the code owner for Experimental IoT patterns
-src/pages/community/patterns/login-pattern/* @lukefirth
```
[the remark this answers is not in the document]
Comment (lukefirth) [reply .github/CODEOWNERS:23]: I am... "Ish". I don't know about the intent of the codeowners lines here though. I'd update:
- Line 22 this to remove reference to 'Experimental IoT patterns' and Chris. I'd reference Sustainability Software Patterns with @dianatran18 (currently on leave) as general owner.
- Line 23 also to Diana, or if it should be someone available, then to Kevin Weaver (unfortunately he's not @'able presently), or myself as final fallback.

(For background: we have a fully fledged variant on the carbon login pattern at the sustainability software level)

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,32 +1,19 @@
...
+# Diana Tran is the owner for Sustainability Software Patterns
 
-# Vikki Paterson is code owner for Experimental IDE patterns
-src/pages/community/patterns/create-flows/* @vikkipaterson
-src/pages/community/patterns/delete-pattern/* @vikkipaterson
-src/pages/community/patterns/export-pattern/* @vikkipaterson
-src/pages/community/patterns/generate-an-api-key/* @vikkipaterson
-src/pages/community/patterns/import-pattern/* @vikkipaterson
-src/pages/community/patterns/remove-pattern/* @vikkipaterson
+src/pages/community/patterns/login-pattern/\*  @dianatran18 @lukefirth
```
Comment (tw15egan) [inline .github/CODEOWNERS:7]:
```suggestion
src/pages/community/patterns/login-pattern/*  @dianatran18 @lukefirth
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,32 +1,19 @@
...
-# Chris Reckling is the code owner for Experimental IoT patterns
-src/pages/community/patterns/login-pattern/* @lukefirth
+# Zvonimir Fras for Angular tutorial
 
-# Eliad Moosavi and Natasha Decoste are the code owners for Data Visualization
-src/pages/data-visualization/* @theiliad @natashadecoste
-
-# Cal Smith for Angular tutorial
-/src/pages/tutorial/angular/* @zvonimirfras
+/src/pages/tutorial/angular/\* @zvonimirfras
```
Comment (tw15egan) [inline .github/CODEOWNERS:15]:
```suggestion
/src/pages/tutorial/angular/* @zvonimirfras
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -1,32 +1,19 @@
...
 
 # Lee Chase for Vue tutorial
-/src/pages/tutorial/vue/* @lee-chase
+
+/src/pages/tutorial/vue/\* @lee-chase
```
Comment (tw15egan) [inline .github/CODEOWNERS:19]:
```suggestion
/src/pages/tutorial/vue/* @lee-chase
```

**Comment by sstrubberg** [comment]
@tw15egan i committed the changes, but it looks like the errors are from the fact that the code owners in question don't have write access to the monorepo

**Review by laurenmrice — APPROVED** [review]

**Review by andreancardona — APPROVED** [review]

**Review by tay1orjones — APPROVED** [review]
@sstrubberg I added all 4 'erroring' users to the [`carbon-design-system/maintainers-website-patterns`](https://github.com/orgs/carbon-design-system/teams/maintainers-website-patterns/members) team, should be good to go

**Review by tw15egan — APPROVED** [review]

**Pull request merged**